src/google76f9c5285f5b21c9.html

```diff
@@ -0,0 +1 @@
+google-site-verification: google76f9c5285f5b21c9.html
```

Comment:
O que é esse arquivo? Se não for necessário, pode remover?

Reply:
Era um arquivo que o GSC pdia para adicionar à home para permitir a indexação do preview, as não deu certo, é pra remover também.